src/Microsoft.VisualStudio.ProjectSystem.Managed/Annotations.cs

```diff
@@ -143,3 +143,10 @@ public MemberNotNullWhenAttribute(bool returnValue, params string[] members)
         public string[] Members { get; }
     }
 }
+
+namespace System.Runtime.CompilerServices
+{
+    internal sealed class IsExternalInit
```

Comment:
Consider adding a code comment to emphasize that this class should *always* be present rather than `#if` based on TFM, or else it'll be [a binary breaking change](https://github.com/manuelroemer/IsExternalInit/issues/5).